numpy/__init__.pyi

```diff
@@ -293,84 +399,20 @@ class ndarray(_ArrayOrScalarCommon, Iterable, Sized, Container):
     @shape.setter
     def shape(self, value: _ShapeLike): ...
     @property
-    def flat(self: _ArraySelf) -> flatiter[_ArraySelf]: ...
-    @property
     def strides(self) -> _Shape: ...
     @strides.setter
     def strides(self, value: _ShapeLike): ...
-    # Array conversion
```

Comment:
These methods are present in both `np.ndarray` and `np.generic`.
Note that, as the `_ArraySelf` typevar is bound to the `_ArrayOrScalarCommon` superclass, these methods can be moved without further adjustments.